otterdog/eclipse-velocitas.jsonnet

```diff
@@ -503,5 +503,24 @@ orgs.newOrg('eclipse-velocitas') {
         },
       ],
     },
+    orgs.newRepo('pkg-velocitas-main') {
+      allow_update_branch: false,
+      delete_branch_on_merge: true,
+      has_wiki: false,
+      web_commit_signoff_required: false,
+      workflows+: {
+        actions_can_approve_pull_request_reviews: false,
+        default_workflow_permissions: "write",
```

Comment:
we would like to enforce the default workflow permission to be set to read only.

You can easily specify which permissions each workflow needs: https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
I understand that right now its write for most of the repos, but when creating new repos you should use read only. After time all workflow should explicitly state which permissions they need rather than giving them write tokens by default.